pkgs/applications/window-managers/i3/gaps.nix

```diff
@@ -11,15 +11,14 @@ i3.overrideDerivation (super : rec {
     sha256 = "16s6bink8yj3zix4vww64b745d5drf2vqjg8vz3pwzrark09hfal";
   };
 
-  nativeBuildInputs = super.nativeBuildInputs ++ [ autoreconfHook ];
+  nativeBuildInputs = oldAttrs.nativeBuildInputs ++ [ autoreconfHook ];
 
   postUnpack = ''
       echo -n "${version} (${releaseDate})" > ./i3-${version}/I3_VERSION
   '';
 
   # fatal error: GENERATED_config_enums.h: No such file or directory
   enableParallelBuilding = false;
-}) // {
 
   meta = with stdenv.lib; {
```

Comment:
I don't think meta works with overrideAttrs. Can you confirm that the meta is still being applied? I though you still had to do a union like with overrideDerivation.

Reply:
Does indeed work, the main reason I'm doing this PR is to have `meta.position` available. All other meta attributes work as well.

Reply:
Ok looks good then.